src/evidently/test_suite/test_suite.py

```diff
@@ -154,15 +154,17 @@ def as_dict(self) -> dict:
             renderer = find_test_renderer(type(test), self._inner_suite.context.renderers)
             test_results.append(renderer.render_json(test))
 
+        total_tests = len(self._inner_suite.context.test_results)
+
         return {
             "version": evidently.__version__,
             "datetime": datetime.now().isoformat(),
             "tests": test_results,
             "summary": {
                 "all_passed": bool(self),
-                "total_tests": len(self._inner_suite.context.test_results),
-                "success_tests": len(self._inner_suite.context.test_results),
-                "failed_tests": len(self._inner_suite.context.test_results),
+                "total_tests": total_tests,
```

Comment:
wouldn't be more robust to insert `counter["SUCCESS"] + counter["WARNING"] + counter["FAIL"]`?

Reply:
we have SKIPPED and ERROR statuses - total tests can be greater than `counter["SUCCESS"] + counter["WARNING"] + counter["FAIL"]`